.eslintrc.js

```diff
@@ -18,16 +18,44 @@ module.exports = {
         'react/no-access-state-in-setstate': 'off', // fixme
         'react/no-array-index-key': 'off', // fixme
         'react/no-this-in-sfc': 'off',
+        'import/no-unresolved': 'off', // fixme
     },
     plugins: ['formatjs'],
     overrides: [
         {
-            files: ['*.test.js'],
+            files: ['*.test.js', '*.test.tsx'],
             globals: {
                 shallow: true,
                 mount: true,
                 takeScreenshot: true,
             },
+        },
+        {
+            files: ['*.ts', '*.tsx'],
+            rules: {
+                '@typescript-eslint/explicit-function-return-type': 'off', // fixme
+            },
+            parser: "@typescript-eslint/parser",
+            extends: [
+                eslintrc,
+                "plugin:@typescript-eslint/recommended",
+                "prettier/@typescript-eslint",
+            ],
+            "settings": {
+                "import/resolver": {
+                "typescript": {
+                    "directory": "."
+                }
+                }
+            },
+            overrides: [
+                {
+                    files: ['*.stories.tsx', '*.test.tsx'],
+                    rules: {
+                        'import/no-extraneous-dependencies': 'off'
```

Comment:
we dont want to consider stories as something needed at runtime. So imports dont need to be in deps section and can remain in dev deps.